services/identity-service/build/Jenkinsfile

```diff
@@ -0,0 +1,45 @@
+node() {
```

Comment:
Should this be present within the repo. I think this can be removed.

Reply:
the file has been resolved as per discussion during the standup